From c3ddd27453916ff7ece4ab82d4722233d7ab4074 Mon Sep 17 00:00:00 2001
From: Paramahansa Polo <ppolo@habana.ai>
Date: Fri, 29 May 2020 16:16:44 +0300
Subject: [PATCH] [SW-13556] robot: (en/dis)-able pcie_reboot test

A stress test has been added to robot in order to check PCIe links
after soft reboots ( see jita ticket SW-10973). However, that test
takes too long and it might not be needed for certain test runs.
Here we add the option to enable disable the "pcie_reboot" test.

Change-Id: I31088f5e2bd141e3b071ee64a011a47351565fae
---

diff --git a/package/robot/robot.mk b/package/robot/robot.mk
index 4d497f0..e03c379 100644
--- a/package/robot/robot.mk
+++ b/package/robot/robot.mk
@@ -10,9 +10,15 @@
 	LTP_TESTSUITE = 1
 endif
 
+PCIE_REBOOT = 0
+ifeq ($(RUN_PCIE_REBOOT),yes)
+	PCIE_REBOOT = 1
+endif
+
 define ROBOT_BUILD_CMDS
 	(cd $(@D); 							\
 		robot	-v ltp-testsuite:$(LTP_TESTSUITE)		\
+			-v run_pcie_reboot:$(PCIE_REBOOT)		\
 			-v HL_INSTALL_HOST_IP:$(HL_INSTALL_HOST_IP) 	\
 			-v HL_INSTALL_LAB_IP:$(HL_INSTALL_LAB_IP)	\
 			-v PDU_PORT:$(PDU_PORT)				\
